h/templates/client/annotation.html

```diff
@@ -45,11 +45,13 @@
 <section class="annotation-section"
          ng-repeat="target in vm.annotation.target track by $index">
   <blockquote class="annotation-quote"
+              excerpt=""
```

Comment:
When directives take no arguments and are tightly coupled to styling it might just make sense to restrict them to classes instead and to invoke them by applying the class to the element rather than an attribute.